### DISCUSSION TITLE
월드 빌드잇

### DISCUSSION THREAD

**Discussion by sabalmyeon** [naverz/zepeto-studio-kor#509] [4. General]
빌드잇에서 만든것을 유니티로 불러와 월드로 만들면 빌드잇에서 상호작용되던것 (차타기, 앉기 등등)은 없어지고 오브젝트들만 있나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 말씀하신 대로 상호작용 오브젝트는 작동하지 않을 수 있습니다.

아래 내용을 참고하여 주세요.
https://studio.zepeto.me/kr/guides/loading-buildit-maps
